### PR TITLE
lexical-localization

### PR DESCRIPTION
**Pull request by tagcode**
Adds localization class library.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -417,6 +417,7 @@ Check out my [blog](https://medium.com/@thangchung) or say hi on [Twitter](https
 * [Localization](https://github.com/aspnet/Localization) - Localization abstractions and implementations for ASP.NET Core applications.
 * [NetCoreStack.Localization](https://github.com/NetCoreStack/Localization) - Database Resource Localization for .NET Core with Entity Framework and In Memory Cache
 * [Westwind.Globalization](https://github.com/RickStrahl/Westwind.Globalization) - Database driven resource localization for .NET applications.
+* [Lexical.Localization](https://github.com/tagcode/Lexical.Localization) - Localization class library for .NET.
```
Comment (ngohungphuc) [inline README.md]: Please order into alphabet order

Comment (tagcode) [reply README.md]: Sure

**Comment by tagcode** [comment]
Bump?

**Comment by Jogai** [comment]
lgtm

**Pull request closed without merging**